daemon/network.go

```diff
@@ -358,6 +317,15 @@ func (daemon *Daemon) createNetwork(create types.NetworkCreateRequest, id string
 		nwOptions = append(nwOptions, libnetwork.NetworkOptionConfigFrom(create.ConfigFrom.Network))
 	}
 
+	if agent && driver == "overlay" && (create.Ingress || runtime.GOOS == "windows") {
+		nodeIP, exists := daemon.GetAttachmentStore().GetIPForNetwork(id)
+		if !exists {
+			return nil, fmt.Errorf("Failed to find a load balancer IP to use for network: %v", id)
```

Comment:
we should consider also the case where the error is associated with the ingress network else this message can be misleading

Reply:
actually considering that the ingress is actually an instance of a load balancer sandbox, the id should give enough information on what to look for